docs/topics/gradle/gradle-configure.md

```diff
@@ -0,0 +1,745 @@
+            }
+        }
+    }
+}
+```
+
+</tab>
+</tabs>
+
+Alternatively, you can [set dependencies at the top level](#set-dependencies-at-the-top-level).
+
+### Dependency types
+
+Choose the dependency type based on your requirements.
+
+<table>
+    <tr>
+        <th>Type</th>
+        <th>Description</th>
+        <th>When to use</th>
+    </tr>
+    <tr>
+        <td><code>api</code></td>
+        <td>Used both during compilation and at runtime and is exported to library consumers.</td>
+        <td>If any type from a dependency is used in the public API of the current module, use an <code>api</code> dependency.
+        </td>
+    </tr>
+    <tr>
+        <td><code>implementation</code></td>
+        <td>Used during compilation and at runtime for the current module, but is not exposed for compilation of other modules
+            depending on the one with the `implementation` dependency.</td>
+        <td>
+            <p>Use for dependencies needed for the internal logic of a module.</p>
+            <p>If a module is an endpoint application which is not published, use <code>implementation</code> dependencies instead
+                of <code>api</code> dependencies.</p>
+        </td>
+    </tr>
+    <tr>
+        <td><code>compileOnly</code></td>
+        <td>Used for compilation of the current module and is not available at runtime nor during compilation of other modules.</td>
+        <td>Use for APIs which have a third-party implementation available at runtime.</td>
+    </tr>
+    <tr>
+        <td><code>runtimeOnly</code></td>
+        <td>Available at runtime but is not visible during compilation of any module.</td>
+        <td></td>
+    </tr>
+</table>
+
+### Dependency on the standard library
+
+A dependency on the standard library (`stdlib`) is added automatically to each source set. The version
+of the standard library used is the same as the version of the Kotlin Gradle plugin.
+
+For platform-specific source sets, the corresponding platform-specific variant of the library is used, while a common standard
+library is added to the rest. The Kotlin Gradle plugin will select the appropriate JVM standard library depending on
+the `kotlinOptions.jvmTarget` [compiler option](gradle-compiler-options.md) of your Gradle build script.
+
+If you declare a standard library dependency explicitly (for example, if you need a different version), the Kotlin Gradle
+plugin won't override it or add a second standard library.
+
+If you do not need a standard library at all, you can add the opt-out option to the `gradle.properties`:
+
+```kotlin
+kotlin.stdlib.default.dependency=false
+```
+
+### Set dependencies on test libraries
+
+The [`kotlin.test`](https://kotlinlang.org/api/latest/kotlin.test/) API is available for testing Kotlin projects on
+all supported platforms.
+Add the dependency `kotlin-test` to the `commonTest` source set, and the Gradle plugin will infer the corresponding
```

Comment:
```suggestion
Add the dependency `kotlin-test` to the `commonTest` source set, so that the Gradle plugin can infer the corresponding
```

Reply:
It's not the possibility, it will infer:)

Reply:
I'm trying to avoid future tense 😄 What I mean in this sentence is that only by adding the dependency **can** the Gradle plugin infer. But maybe I misunderstand how this works?

Reply:
And what's the problem with the future tense? We have 740 occurrences of the word "will" in the project😅

Reply:
In my last company, we largely followed Google's developer documentation style guide where the goal is to use the present tense as much as possible. Here is the relevant page: https://developers.google.com/style/tense But if that's not our style, let me know 😄

Reply:
Yes, it's a rule to escape future tense in most of the companies (TBH, in every company I've worked for). Google developer docs style guide is a good reference with an explanation. 

Our docs were written for years with a lot of different people, so we don't have such rule, and it will take a while to rephrase all these lines in our docs. Let's keep it simple and omit such cases.